dora/tests/src/test/java/alluxio/testutils/IntegrationTestUtils.java

```diff
@@ -105,36 +100,6 @@ public static void waitForFileCached(final FileSystem fileSystem, final AlluxioU
     }, WaitForOptions.defaults().setTimeoutMs(timeoutMs));
   }
 
-  /**
-   * Triggers two heartbeats to wait for a given list of blocks to be removed from both master and
-   * worker.
-   * Blocks until the master and block are in sync with the state of the blocks.
-   *
-   * @param bw the block worker that will remove the blocks
-   * @param blockIds a list of blockIds to be removed
-   */
-  public static void waitForBlocksToBeFreed(final BlockWorker bw, final Long... blockIds)
```

Comment:
I had to remove this utility because it depends on the removed `BlockHeartbeatReporter`. Luckily all callers have also been removed.